Regenerate enterprises and configuration (#678)
MewsSystems/gitbook-connector-api: operations/enterprises.md

<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
# Enterprises

## Get all enterprises

> Returns all enterprises within scope of the `Access Token`, optionally filtered by enterprise identifiers and external identifiers. This operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).

Reviewer: Are we dropping the reference to pagination now?

Author: Added back, we're missing in on few endpoints 😅

      "Pricing": "Gross",
      "TaxPrecision": 2,
      "AddressId": "31c505e9-9858-4d2f-9eab-afce011c4f47",
      "GroupNames": [
        "Sample Group Name"
      ],
      "Subscription": {
        "TaxIdentifier": "123456789 RC 0001"
      }
    }
  ],
  "Cursor": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
}
```

| Property | Type | Contract | Description |
| :-- | :-- | :-- | :-- |
| `Enterprises` | array of [Enterprise](enterprises.md#enterprise) | required | The filtered enterprises. |
| `Cursor` | string | optional | Unique identifier of the last and hence oldest enterprise returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older enterprises. |

#### Enterprise

| Property | Type | Contract | Description |
| :-- | :-- | :-- | :-- |
| `Id` | string | required | Unique identifier of the enterprise. |
| `ExternalIdentifier` | string | optional, max length 255 characters | Identifier of the enterprise from external system. |

> | `ChainId` | string | required | Unique identifier of the chain whose member the enterprise is. |

Reviewer: I prefer the original version of the description
```suggestion
| `ChainId` | string | required | Unique identifier of the chain to which the enterprise belongs. |
```
